examples/decoding/plot_decoding_xdawn_meg.py

```diff
@@ -0,0 +1,101 @@
+"""
+=============================
+ XDAWN Decoding From MEG data
+=============================
+
+ERF decoding with Xdawn. For each event type, a set of spatial Xdawn filters
+are trained and apply on the signal. Channels are concatenated and rescaled to
+create features vectors that will be fed into a Logistic Regression.
+
+References
+----------
+[1] Rivet, B., Souloumiac, A., Attina, V., & Gibert, G. (2009). xDAWN
+algorithm to enhance evoked potentials: application to brain-computer
+interface. Biomedical Engineering, IEEE Transactions on, 56(8), 2035-2043.
+
+[2] Rivet, B., Cecotti, H., Souloumiac, A., Maby, E., & Mattout, J. (2011,
+August). Theoretical analysis of xDAWN algorithm: application to an
+efficient sensor selection in a P300 BCI. In Signal Processing Conference,
+2011 19th European (pp. 1382-1386). IEEE.
+"""
+# Authors: Alexandre Barachant <alexandre.barachant@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from sklearn.cross_validation import StratifiedKFold
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import classification_report, confusion_matrix
+from sklearn.preprocessing import MinMaxScaler
+
+from mne import io, pick_types, read_events, Epochs
+from mne.datasets import sample
+from mne.preprocessing import Xdawn
+from mne.decoding import ConcatenateChannels
+
+
+print(__doc__)
+
+data_path = sample.data_path()
+
+###############################################################################
+# Set parameters and read data
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+event_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw-eve.fif'
+tmin, tmax = -0.1, 0.3
+event_id = dict(aud_l=1, aud_r=2, vis_l=3, vis_r=4)
```

Comment:
- aud_l v.s. aud_r  = 82%
- viz_l v.s. viz_r  = 96.5%
- aud_l v.s. viz_l  = 98.6%
- aud_l v.s. viz_r  = 95.7%
- aud_r v.s. viz_l  = 99.3%
- aud_r v.s. viz_r  = 97.2%

the 4 class accuracy is 93.4%